src/main/java/com/github/manolo8/darkbot/core/manager/EffectManager.java

```diff
@@ -60,15 +60,132 @@ public enum Effect {
         UNDEFINED(-1),
         LOCATOR(1),
         PET_SPAWN(2),
+        SOLACE_NANO_CLUSTER(3),
+        DIMINISHER_WEAKEN_SHIELDS(4),
+        SPECTRUM_PRISMATIC_SHIELD(5),
+        SENTINEL_FORTRESS(6),
+        /** Cyborg uses the same effect as venom for singularity */
+        VENOM_SINGULARITY(7),
+        SABOTEUR(9),
+        /* SHIELD_1(10), */
         ENERGY_LEECH(11),
+        REPAIR_BATTLE_BOT(12),
+        /* LEVEL_UP(13), */
+        LIGHTNING_ABILITY(14),
+        HEAL_RAY(15), // AEGIS_HP_REPAIR RAY
         NPC_ISH(16),
-        DRAW_FIRE(36),
```

Comment:
Sadly due to backwards compatibility this one must stay, position it right next to the new "CITADEL_DRAW_FIRE_VICTIM", keep both, and just `@Deprecate` this one